worker/provisioner/provisioner.go

```diff
@@ -405,5 +405,13 @@ func (p *containerProvisioner) getRetryWatcher() (watcher.NotifyWatcher, error)
 }
 
 func (p *containerProvisioner) getProfileWatcher() (watcher.StringsWatcher, error) {
-	return p.st.WatchModelMachinesCharmProfiles()
+	if p.containerType != instance.LXD {
+		// TODO (hml) lxd-profile 17-oct-2018
+		// what is the correct way to handle this, only start for LXD containers
```

Comment:
I wonder if we can return `nil, nil` or a dumb watcher, that never watches anything, so effectively it becomes a no-op watcher?